hbase-it/src/test/java/org/apache/hadoop/hbase/chaos/factories/ServerAndDependenciesKillingMonkeyFactory.java

```diff
@@ -83,5 +84,8 @@ private void loadProperties() {
     rollingBatchSuspendtRSRatio = Float.parseFloat(this.properties.getProperty(
         MonkeyConstants.ROLLING_BATCH_SUSPEND_RS_RATIO,
         MonkeyConstants.DEFAULT_ROLLING_BATCH_SUSPEND_RS_RATIO + ""));
+    killMetaRs = Boolean.parseBoolean(this.properties.getProperty(
+      MonkeyConstants.KILL_META_RS,
+      MonkeyConstants.DEFAULT_KILL_META_RS + ""));
```

Comment:
Similar suggestion as done for the Actions. Can we move this to a common method in the parent class to avoid code repetition?